KR_Pricing/KR_Pricing_Web/HttpService.cs

```diff
@@ -14,10 +13,10 @@ public class HttpService : IHttpService
         private HttpClient _httpClient;
         private NavigationManager _navigationManager;
 
-        public HttpService(IHttpClientFactory clientFactory, NavigationManager navigationManager)
+        public HttpService(HttpClient httpClient ,NavigationManager navigationManager)
```

Comment:
We shouldn't change existing codebase here
```suggestion
        public HttpService(IHttpClientFactory clientFactory ,NavigationManager navigationManager)
```